rlog/2024-06-24-gsub-largemsg-improvements.mdx

```diff
@@ -0,0 +1,341 @@
+---
+title: 'Large Message Handling in GossipSub: Potential Improvements'
+date: 2024-06-24 15:00:00
+authors: farooq
+published: false
+slug: GossipSub Improvements
+categories: research
+
+
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+
+Large Message Handling in GossipSub: Potential Improvements 
+
+<!--truncate-->
+
+## Motivation:
+The challenge of large message transmissions in gossipsub leads to longer than expected network-wide message dissemination times (and relatively higher fluctuations). 

```

Comment:
Gossipsub probably should be capitalized at every instance as it is a proper noun.

Reply:
bd3f3d9c0f32350e07e987ff0ffc46ed9e3ba174